packages/lesswrong/components/posts/PostsAnalyticsPage.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { Components, registerComponent } from '../../lib/vulcan-lib'
+import { useLocation, useServerRequestStatus } from '../../lib/routeUtil'
+import { useSingle } from '../../lib/crud/withSingle'
+import { useCurrentUser } from '../common/withUser'
+import { userOwns } from '../../lib/vulcan-users'
+import { usePostAnalytics } from './usePostAnalytics'
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { Link } from '../../lib/reactRouterWrapper'
+import { forumTypeSetting } from '../../lib/instanceSettings'
+import NoSsr from '@material-ui/core/NoSsr';
+import classNames from 'classnames';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  viewingNotice: {
+    marginTop: theme.spacing.unit * 4,
+    '& a': {
+      color: theme.palette.primary.main,
+    },
+  },
+  // TODO: right now on lw-master, gutterBottom exists on the Typography class,
+  // but we don't yet have that code
+  gutterBottom: {
+    marginBottom: '0.35em',
+  },
+  calculating: {
+    marginLeft: theme.spacing.unit * 2,
+  },
+})
+
+const PostsAnalyticsInner = ({ classes, post }) => {
+  const { postAnalytics, loading, error } = usePostAnalytics(post._id)
+  const { Loading, Typography } = Components
+  if (loading) {
+    return <>
+      <Typography variant="body1" className={classNames(classes.gutterBottom, classes.calculating)}>
+        <em>Calculating metrics. (This can take some time for popular posts.)</em>
+      </Typography>
+      <Loading />
+    </>
+  }
+  if (error) {
+    throw error
+  }
+
+  return (<Table>
```

Comment:
![image](https://user-images.githubusercontent.com/25752873/134868535-72daf4a3-e3ff-4903-a9ae-4c3aef453284.png)

This looks a bit cramped on mobile. Layout.tsx switches from 50px top padding to 0px at the 'sm' breakpoint, which is most of the problem. I think adding top and side padding would both help.

Related, I feel like that's a weird default behavior for Layout.tsx to have. I feel like the default padding should be some sensible nonzero value, and any pages that need 0 top padding should use negative padding to compensate. Not a blocking issue for this PR though.